walkthroughs/week-3-server/examples/server-stats/src/main/webapp/index.html

```diff
@@ -3,8 +3,8 @@
   <head>
     <meta charset="UTF-8">
     <title>Server Stats</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link rel="stylesheet" href="./style.css">
```

Comment:
Same comment here as for the script.js in the walkthroughs folder; it doesn't affect your site, but it's also probably not intentional. Go ahead and revert the changes in this file if you want.